Library/Homebrew/formula_auditor.rb

```diff
@@ -527,6 +533,7 @@ def audit_homepage
       return unless @online
 
       return if formula.tap&.audit_exception :cert_error_allowlist, formula.name, homepage
+      return if formula.tap&.audit_exception :github_ip_not_permitted_allowlist, formula.name, homepage
```

Comment:
Maybe we should allow this error unconditionally instead of relying on an allowlist?

Reply:
I was under the wrong impression that the failed audit is `audit_homepage`, but it's rather another one (we don't use GitHub API while checking the homepage, but the error message mentions `GitHub API Error`). 
I suspect it's one of the other audits that calls GitHub API, so I've moved the logic to `SharedAudits`.